go.mod

```diff
@@ -153,6 +153,7 @@ require (
 )
 
 replace (
+	github.com/openshift/api => github.com/flavianmissi/api v0.0.0-20220901145017-956aca874f9f
```

Comment:
waiting on https://github.com/openshift/openshift-controller-manager/pull/244 to fix this